README.md

```diff
@@ -16,6 +17,18 @@ $ ./gradlew installDist
 $ build/install/scalar-admin/bin/scalar-admin
 ```
 
```

Comment:
Why don't you add usage for the container? Most users would just use it instead of creating it, right?

Reply:
> Why don't you add usage for the container? 

Sorry for the inconvenience. I left to add. 
Steps for container usage added.

> Most users would just use it instead of creating it, right?

yes